eisbuk-admin/src/components/atoms/AttendanceCard/__tests__/AttendanceCard.test.tsx

```diff
@@ -54,10 +68,61 @@ describe("AttendanceCard", () => {
     });
     test("should disable attendance button while state and fb are synching", () => {
       screen.getByText("👎").click();
-      expect(screen.getByTestId("SaulGoodman")).toHaveProperty(
+      expect(screen.getByTestId("Saul-Goodman")).toHaveProperty(
         "disabled",
         true
       );
     });
+    /** @TODO find a good test for options */
+    test("should render intervals in dropdown when athlete is present", () => {
+      screen.getByText("👎").click();
+      expect(screen.getByTestId("Saul-Goodman-input")).toBeInTheDocument();
```

Comment:
There's no need for explicit assertion here `.toBeInTheDocument()` as `getByTestId()` will throw an error if test id isn't found within the document.